plugins/populator/frontend/public/javascripts/countly.models.js

```diff
@@ -805,14 +888,18 @@
         });
     }
 
+    /**
+    * Generate retentions
+    * @param {callback} callback - callback function
+    **/    
     function generateRetention(callback) {
         if (typeof countlyRetention === "undefined") {
             callback();
             return;
         }
         var ts = endTs - 60 * 60 * 24 * 9;
         var ids = [ts];
-        var users = 10;
+        users = 10;
```

Comment:
This approach is very dangerous, with var it would create local function scope variable and be ok with it,
but without var, it would change the upper scope variable, change its value and lead to unpredictable results